### PR TITLE
Change component definitions

### PR DESCRIPTION
**Pull request by ascrutae**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

### CODE REVIEW

**Comment by ascrutae** [comment]
I will change the test case later and run it.

File: apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml [hunk 1]
```diff
@@ -123,7 +123,36 @@ Jedis:
...
+Spymemcached:
+  id: 35
+  languages: Java
+Xmemcached:
+  id: 36
+  languages: Java
+postgresql-jdbc-driver:
+  id: 37
+  languages: Java
+rocketMQ-producer-client:
```
Comment (wu-sheng) [inline apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml]: producer and client are same thing for kafka

Comment (wu-sheng) [reply apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml]: or rocketmq

File: apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java [hunk 2]
```diff
@@ -123,6 +147,18 @@ public ComponentsDefine() {
...
+        addComponent(OJDBC);
+        addComponent(JEDIS);
+        addComponent(SPYMEMCACHED);
+        addComponent(XMEMCACHED);
+        addComponent(POSTGRESQL_DRIVER);
+        addComponent(ROCKET_MQ_PRODUCER);
+        addComponent(ROCKET_MQ_CONSUMER);
+        addComponent(KAFKA_MQ_PRODUCER);
+        addComponent(KAFKA_MQ_CONSUMER);
+        addComponent(MONGO_DRIVER);
```
Comment (wu-sheng) [inline apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java]: Why add all  components as new IDs? What about the old ones?

File: apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml (hunk 1 above)
Comment (ascrutae) [reply apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml]: well, How can I mapping the kafka or rocketMQ in the `Component-Server-Mappings` node?

Comment (wu-sheng) [reply apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml]: you should name as consumer and producer. No client needed. They are all clients for kafka server.

**Comment by wu-sheng** [comment]
@ascrutae Your CI fails. Please recheck.

**Comment by wu-sheng** [comment]
@ascrutae Please provide screenshot about #1177 issues.

**Comment by wu-sheng** [comment]
You missed the screenshot of the overview tab.

**Comment by ascrutae** [comment]
Here is overview 
wx20180527-114702 2x

**Comment by ascrutae** [comment]
wx20180527-115027 2x

**Pull request merged**